setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
That does not seem to help, I think that the issue is that the invenio-db is already specified in the install_requires, but only with `[versioning]`, and that supersedes the `[versioning,mysql]` that is specified in the extras, never installing the special invenio-db with mysql deps (or postgres). Looking